package.json

```
@@ -1,17 +1,16 @@
 {
-  "name": "@brc-dd/nuxt-i18n",
+  "name": "@globalbrain/nuxt-i18n",
   "type": "module",
   "version": "1.0.20",
-  "packageManager": "pnpm@8.10.5",
+  "packageManager": "pnpm@8.15.4",
   "description": "Minimal module for i18n with locale auto-imports & localized routing",
-  "author": "Johann Schopplich <pkg@johannschopplich.com>",
+  "author": "Kia Ishii <ka.ishii@globalbrains.com>",
```

I think it should be: 

```suggestion
  "author": "Divyansh Singh <divyansh@globalbrains.com>",
```

@brc-dd ☝️ Please check this credit is correct (like email address

---

Yeah, lol 😅